.github/workflows/swift.yml

```
@@ -9,7 +9,7 @@ env:
 jobs:
   lint:
     name: Lint Swift Code
-    runs-on: macos-15
+    runs-on: self-hosted
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)